src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -3487,6 +3725,7 @@ namespace HVACUnitarySystem {
                     }
                     if (UnitarySystem(UnitarySysNum).Humidistat && UnitarySystem(UnitarySysNum).ControlType == LoadBased) {
                         for (HStatZoneNum = 1; HStatZoneNum <= NumHumidityControlZones; ++HStatZoneNum) {
+                            if (UnitarySystem(UnitarySysNum).ControlZoneNum == 0) continue;
```

Comment:
Cycle here on ControlZoneNum == 0 since a zone can be found when ActualZoneNum = 0 and ControlZoneNum = 0.